internal/manifests/internal/gateway/gateway-observatorium.rego

```diff
@@ -0,0 +1,24 @@
+package observatorium
```

Comment:
Suggest to eliminate references to `observatorium` and rename to `lokistack-gateway`. Please doublecheck if the package names in rego can include a dash or else we can go here with `lokistack` as package name.